ui/lib/ldap/addon/components/page/role/create-and-edit.hbs

```diff
@@ -0,0 +1,76 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{if @model.isNew "Create Role" "Edit Role"}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<hr class="is-marginless has-background-gray-200" />
+
+<form {{on "submit" (perform this.save)}} class="has-top-margin-m">
+  <MessageError @errorMessage={{this.error}} />
+
+  <label class="is-label">
+    Role type
+  </label>
+  <Hds::Form::RadioCard::Group @name="role type options" class="has-bottom-margin-m" as |RadioGroup|>
```

Comment:
Yes! I did make tickets and left a comment regarding `<Hds::Form::RadioCard::Group>` since `<Hds::Form:RadioCard>` can be used on its own but it seems like we should consider it the same adoption (you wouldn't use the group component with the cards).